DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py

```diff
@@ -114,10 +114,6 @@
 # process.dqmSaverPB.tag         = 'OnlineBeamMonitor'
```

Comment:
(The comment is not on this line specifically, but it applies to this file.)

This DQM config and the HLT menu used online both include an instance of the plugin `OnlineBeamMonitor`. Does the latter need any updates in light of this PR ?

Reply:
I think so.
Who produces this product now

https://github.com/cms-sw/cmssw/blob/365705add198add5f751c863f47a182ecab0b137/DQM/BeamMonitor/plugins/OnlineBeamMonitor.cc#L279

?

Reply:
So , as I was writing before, both at HLT and in DQM that module is run. In DQM the configuration may still have the ESProducer, at HLT if nobody has changed the menu in confDB the ESProducer is still present.
Of course it would be better to modify the code of the OnlineBeamMonitor.cc to access the beam spot through the EDProducer.

Reply:
> at HLT if nobody has changed the menu in confDB the ESProducer is still present.

but this assumption is incorrect. The customization function here (proposed in this very PR) removes it:

https://github.com/cms-sw/cmssw/blob/fa5f00f1c49c6e378ac47831342a42983b3a5eb7/HLTrigger/Configuration/python/customizeHLTforCMSSW.py#L161-L162

Reply:
I didn't actually think about the DQM monitoring plots. I can leave it in the menu if it is needed.

Reply:
>  I can leave it in the menu if it is needed.

I'd rather remove it and change the monitoring module, such that we monitor the same mechanism we use when running the HLT.

Reply:
the online beam monitor has been updated to fetch information from the hlt collection instead of the transient record. This is done at the first event of each lumisection.